evap/staff/templates/staff_evaluation_form.html

```diff
@@ -19,7 +19,7 @@ <h3>
             {% include 'course_badges.html' with course=evaluation.course %}
         </div>
     {% endif %}
-    <form id="evaluation-form" method="POST" class="form-horizontal multiselect-form select2form">
```

Comment:
When selecting entries, the color is very bold compared to our palette:

![image](https://user-images.githubusercontent.com/47574893/173400153-5985f348-7d2c-4dc7-b48e-22084f10488e.png)
@janno42 what do you think?

Reply:
i think we should redefine the colors to better match our design

Reply:
The color they specify here, #0d6efd, seems to be the bootstrap 5 "active item" default, as it is expanded from `$component-active-bg` here:
https://github.com/orchidjs/tom-select/blob/e466189df876a061df38740c8c02e95daf8dbdbc/src/scss/tom-select.bootstrap5.scss#L24

This is compiled to the CSS rule:
```css
.ts-wrapper.multi .ts-control>div.active {
    background:#0d6efd;
    /// [...]
}
 ```

So it seems we would have to override this. What color would be like? The bg-primary background?

It seems like we use `bg-primary` as defined by bootstrap right now, so I'm a bit confused why the colors mismatch. SCSS can be so confusing...

Reply:
please make adjustments (in `static/scss/_adjustments.scss`) to the selection and box-shadow colors to match the current evap colors